test/jestHelpers.ts

```diff
@@ -0,0 +1,55 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { EOL, platform } from 'os';
+import { describe, expect, test } from '@jest/globals';
```

Comment:
Separated these helpers out because they use `@jest/globals` and importing it isn't allowed in the test runner (index.ts).